src/sgl/refl/function.cpp

```
@@ -0,0 +1,267 @@
+        }
+        m_cached_overloads = std::move(overloads);
+    }
+    return m_cached_overloads.value();
+}
+
+bool Function::is_constructor() const
+{
+    return m_full_name.starts_with("$init");
+}
+
+ref<Function> Function::specialize_with_arg_types(const std::vector<ref<Type>>& types) const
```

Cppcheck (2.20.0)

[style] The function 'zyyx' is never used.

(unusedFunction)